cpp/configs/analysis_example.cfg

```diff
@@ -229,8 +229,10 @@ nnRandomize = true
 # The same as chosenMoveSubtract but only prunes moves that fall below the threshold, does not affect moves above
 # chosenMovePrune = 1
 
-# Number of symmetries to sample (WITH replacement) and average at the root
+# Number of symmetries to sample (WITHOUT replacement) and average at the root
 # rootNumSymmetriesToSample = 1
+# Number of symmetries to sample (WITHOUT replacement) and average at interior nodes
```

Comment:
this slows down the search by this factor, right? should it warn about the effect?

Reply:
Yes but it's the same as rootNumSymmetriesToSample which is already in the codebase and doesn't provide a warning. I can add comments about that if you think it's needed. It all defaults to '1' so no effect unless someone intentionally overrides it.

Reply:
Am i understanding correctly that If you ask for 500 visits, with rootsymmetries=4 you'd take 503 (?) nn evals, and with nodesymmetries=4 you'd take 2000 ?

Reply:
Mentally I was thinking rootsymmetries=4 meant the children of the root, but I guess it actually refers to the actual single root node itself. So yes, rootsymmetries=4 would be 503 nn evals in that case. However the symmetry evals might be counted individually, and so we'd still only do 500 total, with 3 less nodes explored.

Reply:
I added a performance impact warning